sdk/Lusid.Sdk.Tests/ApiClientTests.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Net;
+using Lusid.Sdk.Client;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using Lusid.Sdk.Utilities;
+using Moq;
+using NUnit.Framework;
+using RestSharp;
+
+namespace Lusid.Sdk.Tests
+{
+    [TestFixture]
+    public class ApiClientTests
+    {
+        private ILusidApiFactory _apiFactory;
+        private ApiClient _apiClient;
+        private CustomJsonCodec _customJsonCodec;
+
+        [OneTimeSetUp]
+        public void SetUp()
+        {
+            _apiFactory = TestLusidApiFactoryBuilder.CreateApiFactory("secrets.json");
+            _apiClient = new ApiClient();
+            _customJsonCodec = new CustomJsonCodec(_apiClient.SerializerSettings, GlobalConfiguration.Instance);
+        }
+
+        [Test]
+        [TestCase(" ", "API Response Content was invalid: ' '")]
+        [TestCase("", "API Response Content was invalid: ''")]
```

Comment:
use String.Empty?

Reply:
Inputs into [TestCase(..)]  must be constant expressions, cannot be objects or object method calls, so String.Empty will not compile